ilc/tests/helpers.js

```diff
@@ -0,0 +1,88 @@
+const _ = require('lodash');
```

Comment:
I propose to not use lodash if we can do it.
for example, here we can use `deepmerge` package.